From 719516a8640273da414ecd3cf9730dfa7f06793e Mon Sep 17 00:00:00 2001
From: Lucas Russo <lerwys@gmail.com>
Date: Thu, 27 Jul 2017 14:23:54 -0300
Subject: [PATCH] mlm_server_engine.inc: disable setting logfile to stdout

We don't want malamute to output some DEBUG/INFO
messages on logsystem.
---
 src/mlm_server_engine.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mlm_server_engine.inc b/src/mlm_server_engine.inc
index b48a544..75d6cb1 100644
--- a/src/mlm_server_engine.inc
+++ b/src/mlm_server_engine.inc
@@ -1326,7 +1326,7 @@ s_server_config_global (s_server_t *self)
     int background = atoi (
         zconfig_get (self->config, "server/background", "0"));
     if (!background)
-        zsys_set_logstream (stdout);
+        zsys_set_logstream (NULL);
 }
 
 static s_server_t *
-- 
1.8.3.1

